packages/hardware-trezor/src/transformers/txOut.ts

```diff
@@ -0,0 +1,50 @@
+import * as Trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+import { GroupedAddress } from '@cardano-sdk/key-management';
+import { InvalidArgumentError, Transform } from '@cardano-sdk/util';
+import { TrezorTxOutputDestination, TrezorTxTransformerContext } from '../types';
+import { mapTokenMap } from './assets';
+import { paymentKeyPathFromGroupedAddress, stakeKeyPathFromGroupedAddress } from './keyPaths';
+
+const toDestination: Transform<Cardano.TxOut, TrezorTxOutputDestination, TrezorTxTransformerContext> = (
```

Comment:
this function could be simplified by swapping the conditions like this and removing the most of the nesting:
```ts
if (!knownAddress) {
  return {
    address: txOut.address
  };
}
const paymentPath = paymentKeyPathFromGroupedAddress(knownAddress);
const stakingPath = stakeKeyPathFromGroupedAddress(knownAddress);

if (!stakingPath) throw new InvalidArgumentError('txOut', 'Missing staking key key path.');

return {
  addressParameters: {
  addressType: Trezor.CardanoAddressType.BASE,
  path: paymentPath,
  stakingPath
}
…
```

Reply:
Done in [f3c53ef](https://github.com/input-output-hk/cardano-js-sdk/pull/793/commits/f3c53ef6517130a3c1a8525a2b6cac4961be9f01)